### PR TITLE
Update INSTALLATION.md

### PR DESCRIPTION
**Pull request by thomas-senechal**

### CODE REVIEW

**Review by Thykof — COMMENTED** [review]
The link https://github.com/massalabs/thyra/releases/latest

redirects to https://github.com/massalabs/station/releases/latest

maybe we can directly put https://github.com/massalabs/thyra/releases/latest in the .md file?

**Comment by peterjah** [comment]
> The link https://github.com/massalabs/thyra/releases/latest
> 
> redirects to https://github.com/massalabs/station/releases/latest
> 
> maybe we can directly put https://github.com/massalabs/thyra/releases/latest in the .md file?

lets use station everywhere now !

File: INSTALLATION.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This guide provides step-by-step instructions for installing **MassaStation** on
 
 ## Windows Installation
 
-1. Download the latest version of **MassaStation** installer for Windows (`.msi`) from the [official website](https://github.com/massalabs/thyra/releases/latest/download/massastation-installer_windows_amd64.msi).
+1. Download the latest version of **MassaStation** installer for Windows (`.msi`) from [here](https://github.com/massalabs/thyra/releases/latest/).
```
Comment (Thykof) [inline INSTALLATION.md:17]:
```suggestion
1. Download the latest version of **MassaStation** installer for Windows (`.msi`) from [here](https://github.com/massalabs/station/releases/latest/).
```

File: INSTALLATION.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Congratulations! You have successfully installed **MassaStation** on your comput
 
 ### GUI Installation
 
-1. Download the latest version of **MassaStation** installer for Debian Linux (`.deb`)  from [here](https://github.com/massalabs/thyra/releases/latest/).
+1. Download the latest version of **MassaStation** installer for Debian Linux (`.deb`) from [here](https://github.com/massalabs/thyra/releases/latest/).
```
Comment (Thykof) [inline INSTALLATION.md:43]:
```suggestion
1. Download the latest version of **MassaStation** installer for Debian Linux (`.deb`) from [here](https://github.com/massalabs/station/releases/latest/).
```

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by Thykof — APPROVED** [review]

**Review by B-Naoufal — APPROVED** [review]

**Pull request merged**